### DISCUSSION TITLE
Remove Vertices from Mesh

### DISCUSSION THREAD

**Discussion by wngTn** [facebookresearch/pytorch3d#1189] [General]
Is there a way to remove certain vertices from a mesh that I have obtained through an .obj file?

For instance following this tutorial: https://colab.research.google.com/github/facebookresearch/pytorch3d/blob/stable/docs/tutorials/render_textured_meshes.ipynb#scrollTo=CNndFZxPyjdr

How can I remove some parts of the cow?

**Comment by patricklabatut** [comment]
There is no generic functionality to achieve this in PyTorch3D, although one could leverage the existing mesh [clipping](https://github.com/facebookresearch/pytorch3d/blob/main/pytorch3d/renderer/mesh/clip.py#L323) for specific cases (removing part lying outside of specific bounding planes).
